### PR TITLE
replacing gradle tasks dealing with reporting schema with warehouse s…

### PR DESCRIPTION
**Pull request by rcordeau**
…chema

---
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_admin/5)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -241,12 +241,13 @@ task extractSchema(type: Sync) {
...
-               "cleanReporting": "Cleans the schemas that are required for running locally",
-               "migrateReporting_test": "Migrates the schemas that integration tests depend on",
-               "cleanReporting_test": "Cleans the schemas that integration tests depend on"]
+def taskMap = ["migrateWarehouse": "Migrates the schemas that are required for running locally",
+               "cleanWarehouse": "Cleans the schemas that are required for running locally",
+               "migrateWarehouse_test": "Migrates the schemas that integration tests depend on",
+               "cleanWarehouse_test": "Cleans the schemas that integration tests depend on"]
 
 taskMap.keySet().each { key ->
+    println(key.contains("_test")?"${key.take(key.lastIndexOf('_test'))}Test":"${key}Prod")
```
Comment (malaffoon) [inline build.gradle]: left-over from debugging or intentional?

Comment (rcordeau) [reply build.gradle]: dammit

**Review by malaffoon — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (rcordeau) [reply build.gradle]: deleted.

File: build.gradle [hunk 2]
```diff
@@ -241,10 +241,10 @@ task extractSchema(type: Sync) {
  *********************************************************************/
 
 //These are the tasks available in the Schema dependency that we would want to use in this project
-def taskMap = ["migrateReporting": "Migrates the schemas that are required for running locally",
-               "cleanReporting": "Cleans the schemas that are required for running locally",
-               "migrateReporting_test": "Migrates the schemas that integration tests depend on",
-               "cleanReporting_test": "Cleans the schemas that integration tests depend on"]
+def taskMap = ["migrateWarehouse": "Migrates the schemas that are required for running locally",
+               "cleanWarehouse": "Cleans the schemas that are required for running locally",
+               "migrateWarehouse_test": "Migrates the schemas that integration tests depend on",
```
Comment (bgsavage) [inline build.gradle:246]: I think in reporting isn't it `migrateReportingTest` ?  We should be consistent here.

Comment (bgsavage) [reply build.gradle:246]: Nevermind these are just the tasks

**Pull request merged**